src/theme/custom.scss

```diff
@@ -20,6 +21,7 @@ $alpha: #17172d;
 $primary: #8cbaff;
 $danger: #ff3e3e;
 $success: #1fdb00;
+$modal: #2d2d41;
```

Comment:
We should be able to create this color (or at least a close-enough version of it) by using white with a certain opacity over either `black` or `primary`, see if you can experiment with it and get a close enough color using that method instead of introducing a new theme color here. if it helps, there is another PR open [here](https://github.com/jahorwitz/mu-methods/pull/48) that adds the opacity values